README.md

```diff
@@ -1,43 +1,466 @@
+ */
+public class Main {
+  String rest_endpoint;
+
+  public static void main(String[] args){
+    System.out.println("Hello world!");
+    rest_endpoint = "YOUR_REST_ENDPOINT_HERE"
+  }
+}
+```
+
+### Emphasize
+
+The `emphasize` command only applies to [formatted](#format)
+output. When the `--format` flag is not used to generate formatted
+output, `emphasize` is discarded completely. When the `--format` flag
+is specified, Bluehawk highlights all lines marked with `emphasize`
+command in the specified markup language output. `emphasize` makes it
+easier to keep the correct lines highlighted when you update code
+samples, because it calculates the highlighted line numbers for you.
+You can use `replace` as either a block command or a
+line command.
+
+Consider the following file:
+
+`Main.java`:
+
+```java
+public class Main {
+  public static void main(String[] args){
+    // :code-block-start: modulo
+    int dividend = 11;
+    int divisor = 3;
+    int modulus = dividend % divisor; // :emphasize:
+    System.out.println(dividend + " % " + divisor + " = " + modulus);
+    // :code-block-end:
+  }
+}
+// :replace-end:
+```
+
+Running the following command:
+
+```
+bluehawk snip Main.java -d . --format=sphynx-rst
 ```
 
-Try out some of the commands and options:
+Produces the following output:
 
-- `help`: Display helpful information about available commands. May be more complete and 
-  up-to-date than this README.
-- `copy --state <state name>`: Copy the given `state name` version of files to destination.
-  When Bluehawk encounters a `state` command (see below), multiple versions of the source file
-  are spawned. Each version removes any code in state commands that are **not**
-  marked with the corresponding state name. This flag determines which version
-  to eventually write to disk.
-- `snip`: Output snippet files only. Can be combined with `--state`.
-- `list commands`: Display a list of available markup commands.
-- `-d or --destination` defines the output location.
+`Main.codeblock.modulo.java.code-block.rst`:
+
+```rst
+.. code-block:: java
+   :emphasize-lines: 3
+
+   int dividend = 11;
+   int divisor = 3;
+   int modulus = dividend % divisor;
+   System.out.println(dividend + " % " + divisor + " = " + modulus);
+```
+
+### Remove
+
+The `remove` command, also aliased as `hide`, removes the spanned
+range from Bluehawk output. `remove` can be helpful for hiding
+assertions and state setup from user-facing code samples.
+You can use `remove` as either a block command or a
+line command.
+
+Consider the following file:
+
+`Main.java`:
+
+```java
+public class Main {
+
+  public static void main(String[] args){
+    // :code-block-start: division
+    int dividend = 11;
+    int divisor = 3;
+    int quotient = dividend / divisor;
+    assert(quotient == 3) // :remove:
+    System.out.println(dividend + " / " + divisor + " = " + quotient);
+    // :code-block-end:
+  }
+}
+// :replace-end:
```

Comment:
Needed?

Reply:
That's how I've always implemented replace -- am I missing something? Bluehawk definitely yells at me if I leave replace-end out.

Reply:
Where's the replace-start?

Btw, why was this conversation marked resolved when it still had an open question?

Reply:
That was an accidental inclusion, removed it.